jenkins/Jenkinsfile.nightly

```
@@ -47,68 +41,34 @@ pipeline {
 	options {
 		timeout(time: 10, unit: 'MINUTES')
 	}
+	environment {
+		ENABLE_HTTP_API='http-version3,http-status'
+		ENABLE_WS_API='rpc-v3,blockchain'
+		LISK_APP_WS='ws://host.docker.internal:7887'
+		ENABLE_TESTING_MODE='true'
+    }
 	stages {
 		stage('Checkout SCM') {
 			when {
 				anyOf {
 					branch 'development'
-					branch 'feature/**'
+					branch 'release/**'
```

This should run against development/all release and feature branches too.
```suggestion
					branch 'release/**'
					branch 'feature/**'
```